libs/SalesforceSDKCore/SalesforceSDKCore/Classes/OAuth/SFOAuthCoordinator.m

```diff
@@ -711,6 +723,33 @@ - (NSURLSession*)session {
     return _session;
 }
 
+- (void)presentBioAuth {
```

Comment:
Isn't that a copy of the method in SFLoginViewController ?
Should we have a single implementation in a helper class?

Reply:
I almost did but looked a bit comical.  The utility would just be a method that makes a LAContext and calls evaluate on it.  There is significantly more code in the blocks the caller would need to supply for success or failure.  I will give it another shot though.